backends/p4tools/testgen/targets/bmv2/test/p4-programs/bmv2_action_run.p4

```diff
@@ -0,0 +1,102 @@
+#include <v1model.p4>
```

Comment:
I bet it wouldn't have been too difficult to make all these test programs part of a separate PR.
But now I have started reviewing this one, let's not change this, or I will have to start from scratch.

Reply:
Yes, but then we would have not known whether the tool executes these program correctly. I will create a PR that contributes them to the testdata repository.

Reply:
On second thought, I am not entirely sure whether it makes sense to include them. They might add unnecessary CI time. They are not testing very unique cases.